SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -294,6 +294,11 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Debug", meta = (MetaClass = "SpatialDebugger"))
 	TSubclassOf<ASpatialDebugger> SpatialDebugger;
 
+	/** Toggle to allow debug checking of invalid data modification on non-auth workers */
+	UPROPERTY(EditAnywhere, config, Category = "Debug")
+	bool bEnableSpatialDataDebugger;
+	FORCEINLINE bool IsSpatialDataDebuggerEnabled() const { return bEnableSpatialDataDebugger; }
```

Comment:
nit - I would not FORCEINLINE unless this was causing performance concerns.

Reply:
This is the same as bSpatialNetworking and we will be calling it every frame so I think it's important for performance reasons.